app/main.py

```diff
@@ -1 +1,24 @@
-# write your code here
+from typing import List, Union
+from app.players.elves.elf_ranger import ElfRanger
+from app.players.elves.druid import Druid
+from app.players.dwarves.dwarf_warrior import DwarfWarrior
+from app.players.dwarves.dwarf_blacksmith import DwarfBlacksmith
```

Comment:
You need to import only Elf & Dwarf and the subclasses will catch up by themselves

Reply:
Really, it’s strange that I didn’t see it myself)